pkg/host-timesync-daemon/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM linuxkit/alpine:630ee558e4869672fae230c78364e367b8ea67a9 AS mirror
```

Comment:
should be `linuxkit/alpine:2e362f4459ba4491655061cccdd2fcc7a2de5eb3` now that #2154 is merged